cron_converter/cron.py

```diff
@@ -127,3 +134,17 @@ def validate(self, date_time_obj: Union[datetime, date]) -> bool:
                 valid.append(True)
 
         return all(valid)
+
+
+if sys.modules.get('pydantic', None):
+    from pydantic import GetPydanticSchema
+    from pydantic_core import core_schema
+
+    CronType = Annotated[
```

Comment:
I realised that `Annotated` requires python>=3.9

There are a couple of ways to solve that.
The best option would be to add `typing-extensions` for python<=3.8
What do you think?